extensions/security-jpa/deployment/src/main/java/io/quarkus/security/jpa/deployment/QuarkusSecurityJpaProcessor.java

```diff
@@ -0,0 +1,339 @@
+import io.quarkus.deployment.builditem.ApplicationIndexBuildItem;
+import io.quarkus.deployment.builditem.CapabilityBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.gizmo.AssignableResultHandle;
+import io.quarkus.gizmo.BranchResult;
+import io.quarkus.gizmo.BytecodeCreator;
+import io.quarkus.gizmo.ClassCreator;
+import io.quarkus.gizmo.MethodCreator;
+import io.quarkus.gizmo.MethodDescriptor;
+import io.quarkus.gizmo.ResultHandle;
+import io.quarkus.panache.common.deployment.PanacheEntityClassesBuildItem;
+import io.quarkus.security.identity.SecurityIdentity;
+import io.quarkus.security.identity.request.UsernamePasswordAuthenticationRequest;
+import io.quarkus.security.jpa.deployment.JpaSecurityDefinition.FieldOrMethod;
+import io.quarkus.security.jpa.runtime.JpaIdentityProvider;
+import io.quarkus.security.jpa.runtime.Password;
+import io.quarkus.security.jpa.runtime.PasswordType;
+import io.quarkus.security.jpa.runtime.Roles;
+import io.quarkus.security.jpa.runtime.RolesValue;
+import io.quarkus.security.jpa.runtime.UserDefinition;
+import io.quarkus.security.jpa.runtime.Username;
+import io.quarkus.security.runtime.QuarkusSecurityIdentity;
+
+class QuarkusSecurityJpaProcessor {
+
+    static final DotName DOTNAME_OBJECT = DotName.createSimple(Object.class.getName());
+
+    private static final DotName DOTNAME_STRING = DotName.createSimple(String.class.getName());
+    private static final DotName DOTNAME_LIST = DotName.createSimple(List.class.getName());
+    private static final DotName DOTNAME_SET = DotName.createSimple(Set.class.getName());
+    private static final DotName DOTNAME_COLLECTION = DotName.createSimple(Collection.class.getName());
+
+    private static final DotName DOTNAME_NATURAL_ID = DotName.createSimple(NaturalId.class.getName());
+
+    private static final DotName DOTNAME_USER_DEFINITION = DotName.createSimple(UserDefinition.class.getName());
+    private static final DotName DOTNAME_USERNAME = DotName.createSimple(Username.class.getName());
+    private static final DotName DOTNAME_PASSWORD = DotName.createSimple(Password.class.getName());
+    private static final DotName DOTNAME_ROLES = DotName.createSimple(Roles.class.getName());
+    private static final DotName DOTNAME_ROLES_VALUE = DotName.createSimple(RolesValue.class.getName());
+
+    @BuildStep
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(FeatureBuildItem.SECURITY_JPA);
+    }
+
+    @BuildStep
+    CapabilityBuildItem capability() {
+        return new CapabilityBuildItem(Capabilities.SECURITY_JPA);
+    }
+
+    @BuildStep
+    void configureJpaAuthConfig(ApplicationIndexBuildItem index,
+            BuildProducer<UnremovableBeanBuildItem> unremovable,
+            BuildProducer<GeneratedBeanBuildItem> beanProducer,
+            List<PanacheEntityClassesBuildItem> panacheEntityClasses) throws Exception {
+
+        // Generate an IdentityProvider if we have a @UserDefinition
```

Comment:
Same in this class about the curly braces. Please configure your IDE once and for all as it's a recurring issue in your PRs.

Reply:
If it's a requirement the formatter should do it.